tests/class/helper/SC_Helper_TaxRule/SC_Helper_TaxRule_getTaxDetailTest.php

```
@@ -366,11 +558,261 @@ public function testGetTaxPerTaxRateWithCeil2()
+        );
+        self::assertSame(
+            array_sum($arrTaxInclusiveTotalByRate) - $discount_total,
+            array_sum(array_column($actual, 'total')),
+            '値引後合計の総和が支払額と一致していません'
+        );
+    }
+
     protected function setUpTaxRule(array $taxs = [])
     {
         $this->objQuery->delete('dtb_tax_rule');
         foreach ($taxs as $key => $item) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)